js/ElementRendererProvider.js

```diff
@@ -154,6 +170,10 @@ export default class ElementRendererProvider {
         divEl.appendChild(borderEl);
       }
 
+      if (config.accessibility && config.accessibility.web) {
```

Comment:
do we need it on both div and label elements?

Reply:
@meirotstein It is not totally clear for me what do you mean? For `TYPES.CHECKBOX` we are adding attributes only to label. For `div` we are adding in case of `TYPES.SECTION` in line 227

Reply:
you are right, my bad - sorry. haven't expanded the code..